src/app/containers/Ad/Amp/index.test.jsx

```diff
@@ -24,9 +23,7 @@ const adWithContext = (slotType, toggleContext) => (
     pathname="/"
   >
     <ServiceContextProvider service="afrique">
-      <ToggleContext.Provider value={toggleContext}>
-        <AmpAd slotType={slotType} />
-      </ToggleContext.Provider>
+      <AmpAd slotType={slotType} />
```

Comment:
So we ignore server toggles, and always render the AMP ad slot, and allow the value of that toggle to be determined on the client in all cases.

Reply:
I believe I have just removed the `ampAds` toggle have I gone too far here? I believe the service based checked is done via the `amp-access` tag, is that correct @tochwill ?

Reply:
I think we originally did both to avoid client side calls for all the services that didn't have ads enabled. But now ads are enabled for way more services aren't they? So there's less benefit to checking this on the server, and you don't now have the issue where cached server responses could cause ads to be hidden when the toggle is actually on.

Reply:
We do still have the main ads toggle @tochwill, it is used in Ads container and all pages ads are rendered:
![image](https://user-images.githubusercontent.com/9645462/109996183-e4648200-7d06-11eb-8fb2-9bbd9c2291e3.png)

So if ads are turned off for a service, nothing renders, we have just removed the amp specific serverside toggle, please correct me if that's wrong?

Reply:
No, you're right! Sorry, I'd misremembered.